manifests/facts.pp

```diff
@@ -19,9 +20,20 @@
   }
 
   if $refresh_interval > 0 and $server {
-    $cron_ensure = "present"
+    if $refresh_type == 'systemd' {
+      $cron_ensure = 'absent'
```

Comment:
I use double quotes generally throughout here, please update for consistency.

Reply:
@ripienaar I updated to use double quotes. The only place I left single quotes was the data types in parameters cause other parameters using `Enum` were using single quotes for the possible parameter values.